### DISCUSSION TITLE
What is the statistic by channel?

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#2099] [General]
Hi. What is the statistic by channel used for?   I am interested in a report to categorize chats by the extension that originated them. For example Twilio, web chat, FBMessenger.  Is there a field that can be set in a table to then use the channels report? Or how could this be done? Thanks

**Comment by remdex** [comment]
If integration is done using incoming webhook it's called channel. Other than that you can just setup different department for different source

**Comment by alonsoac** [comment]
Is there a way to set a default icon for chats that have iwh_id = 0 ?

**Reply by alonsoac** [reply]
This is in dashboard panels.

**Comment by alonsoac** [comment]
There seems to be a problem in lhc_web/design/defaulttheme/tpl/lhchat/chat_tabs/information_rows/subject.tpl.php line 14
When chat.iwh_id is >0 and there is an icon file set in the incoming webhook (not material icon) then it fails.
I don't understand why the subject block would be printing the iwh icon.

**Reply by remdex** [reply]
I don't get what you ate trying to achieve there...

**Reply by alonsoac** [reply]
I started using incoming webhook with an icon which is a file not a material icon.   Then in the right side of the chat view, in the information rows , below the subject, started to appear strange characters. This is because the code there is not handling correctly the icon file name and treating it as if it was a material icon name. This looks bad.

**Reply by remdex** [reply]
You should use material icons...

**Reply by remdex** [reply]
P.s for icon as declared you can use image path there if you have extension and upload icon png you want. If you want pure HTML icon from emoji that's need development which can be done under paid service :)
